### DISCUSSION TITLE
How to use a runtime config within a 404 page?

### DISCUSSION THREAD

**Discussion by andrewfritz86** [vercel/next.js#41426] [Help]
My app needs to access certain runtime config on a 404. However, the 404 page is automatically static rendered and can't access the runtime config. Adding the usual SSR hooks results in a build error. Is there a workaround I am not seeing?

**Comment by wwwdepot** [comment]
See create an advanced error page: https://nextjs.org/docs/advanced-features/custom-error-page#more-advanced-error-page-customizing
